vaadin-grid-flow-parent/vaadin-grid-flow-integration-tests/src/test/java/com/vaadin/flow/component/grid/it/GridViewFilteringIT.java

```diff
@@ -66,17 +65,14 @@ public void scrollToEnd_filter_rowsUpdated() {
     }
 
     private void assertRendereredHeaderCell(GridTHTDElement headerCell,
-            String text, boolean componentRenderer, boolean withSorter) {
+            String text, boolean withSorter) {
 
         String html = headerCell.getInnerHTML();
         if (withSorter) {
             Assert.assertTrue(html.contains("<vaadin-grid-sorter"));
         } else {
             Assert.assertFalse(html.contains("<vaadin-grid-sorter"));
         }
-        if (componentRenderer) {
-            Assert.assertTrue(html.contains("<flow-component-renderer"));
-        }
```

Comment:
Cells do not contain a `flow-component-renderer` tag anymore, remaining assertions should still verify that a component tag is rendered.